### PR TITLE
first draft of jobs posting

### PR DESCRIPTION
**Pull request by christoball**
Ive created a draft of the job posting article. Please review for errors and make suggestions
https://deploy.tnris.org/api/v1/preview/e424b9dd8523ee265ee920f7c82411490ec4acb1/news/2016-06-30/tnris-jobs-seeking-two-new-geo-developers/

front page blurb
jobpost

### CODE REVIEW

**Comment by christoball** [comment]
@geochik @rwade2126 can yall take a look and see if it's alright?

File: content/markdown/news/2016-06-30-tnris-jobs-seeking-two-new-geo-developers.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+template: news.html
+title: "TNRIS Jobs: Seeking Two New Geo-Developers"
+author: TNRIS IS
+thumbnail: static/images/texas-gis-forum/2015/sticky_job_square.jpg
+thumbalt: A yellow post-it note reading now hiring
+abstract:
+    TNRIS has two positions open for developers experienced in coding for online GIS applications and more.
+---
+<p class="lead"><img class="pull-right" src="../../../static/images/texas-gis-forum/2015/sticky_job_square.jpg" alt="A sticky note with handwritten Now Hiring!"> TNRIS, part of the Texas Water Development Board, has two positions open for Systems Analysts. These positions help us in developing our website and online data distribution, as well as the key roles in developing online mapping applications. Visit the <a href="https://www.twdb.texas.gov/jobs/"><strong>TWDB Jobs Page</strong></a> for full information and how to apply.</p>
```
Comment (geochik) [inline content/markdown/news/2016-06-30-tnris-jobs-seeking-two-new-geo-developers.md]: TNRIS, a division of...

File: content/markdown/news/2016-06-30-tnris-jobs-seeking-two-new-geo-developers.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+template: news.html
+title: "TNRIS Jobs: Seeking Two New Geo-Developers"
+author: TNRIS IS
+thumbnail: static/images/texas-gis-forum/2015/sticky_job_square.jpg
+thumbalt: A yellow post-it note reading now hiring
+abstract:
+    TNRIS has two positions open for developers experienced in coding for online GIS applications and more.
```
Comment (geochik) [inline content/markdown/news/2016-06-30-tnris-jobs-seeking-two-new-geo-developers.md]: TNRIS is looking for two creative individuals to join their team of passionate solution seekers for the delivery of geographic information and location solution services.

File: content/markdown/news/2016-06-30-tnris-jobs-seeking-two-new-geo-developers.md (hunk 1 above)
Comment (geochik) [inline content/markdown/news/2016-06-30-tnris-jobs-seeking-two-new-geo-developers.md]: The right individuals will enhance our website, improve our online data distribution functionality, and play a key role in developing online location intelligence mapping applications.

**Comment by geochik** [comment]
@christoball I hope my line comments make sense. Doing this on my phone in the middle of the night. 😒

**Comment by christoball** [comment]
@geochik all looks good to me.

**Pull request merged**